backend/src/main/java/com/eyetrade/backend/controller/CommentController.java

```diff
@@ -0,0 +1,63 @@
+package com.eyetrade.backend.controller;
+
+import com.eyetrade.backend.model.dto.CommentDto;
+import com.eyetrade.backend.model.resource.comment.CommentResource;
+import com.eyetrade.backend.security.JwtResolver;
+import com.eyetrade.backend.service.CommentService;
+import io.swagger.annotations.Api;
+import io.swagger.annotations.ApiOperation;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+import java.util.UUID;
+
+/**
+ * Created by Emir Gökdemir
+ * on 24 Kas 2019
+ */
+
+@Api(value = "Comments", tags = {"Comments of Article or Events"})
+@RestController
+@RequestMapping("/comment_controller")
+public class CommentController {
+
+    @Autowired
+    private JwtResolver jwtResolver;
+
+    @Autowired
+    private CommentService service;
+
+    @ApiOperation(value = "A user can comment on event or article with necessary informations.", response = CommentResource.class)
+    @PostMapping("/post-comment")
```

Comment:
Naming convention is different than /comment_controller endpoint. It doesn't really matter but was it deliberate?

Reply:
Good point, I will fix it